views/js/qtiCreator/helper/itemLoader.js

```diff
@@ -65,6 +65,11 @@ define([
                             loadedItem.setNamespaces(namespaces);
                             loadedItem.setSchemaLocations(qtiSchemaLocation);
 
+                            //add languages list to the item
+                            if (data.languagesList) {
+                                loadedItem.attr('languagesList', data.languagesList);
```

Comment:
please do not use qtiElement.attr() to store data that is not a standard QTI element attribute. For this, you should rather use qtiElement.data() which is used to store metadata associated to the element which does not affect its semantic

Reply:
Ok, but it leads to another problem: `item.data` is not accessible to template it only has access to `item.attributes` object. I'll try to find a way to pass data/read it from there.